packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/BorderRadiiDrawableUtils.java

```diff
@@ -18,7 +18,7 @@ private static float getRadiusForCorner(View view, BorderRadiusProp corner, floa
       return defaultValue;
     }
     Rect bounds = view.getBackground().getBounds();
-    return length.resolve(bounds.width(), bounds.height()).toPixelFromDIP().getHorizontal();
+    return length.resolve(bounds.width());
```

Comment:
TODO: add backward-compatibility

Reply:
Removed `BorderRadiiDrawableUtils` and `ReactNativeUtils` which were unused in 230b13e906.

Reply:
When backporting this PR to v3, we'll need to add a `reactNativeVersionPatch` for BorderRadiiDrawableUtils.java. Luckily, this should be pretty straightforward.